panel/tests/chat/test_langchain.py

```diff
@@ -0,0 +1,33 @@
+import platform
+
+import pytest
+
+from langchain.agents import AgentType, initialize_agent, load_tools
+from langchain.llms.fake import FakeListLLM
```

Comment:
I would like to see these inside a try/except with a skip test. So I can run tests locally without having to install langchain.